From: Ben Hutchings <benh@debian.org>
Date: Mon, 31 May 2021 02:08:53 +0200
Subject: Revert "scsi: fcoe: Fix mismatched fcoe_wwn_from_mac declaration"
Forwarded: not-needed

Revert commit 07391893d3cf3d224cca5fc37d1372cf87a8c985.  It had no
semnatic effect but makes genksyms spuriously detect an ABI change.

---
--- a/include/scsi/libfcoe.h
+++ b/include/scsi/libfcoe.h
@@ -261,7 +261,7 @@ int fcoe_ctlr_recv_flogi(struct fcoe_ctl
 			 struct fc_frame *);
 
 /* libfcoe funcs */
-u64 fcoe_wwn_from_mac(unsigned char mac[MAX_ADDR_LEN], unsigned int, unsigned int);
+u64 fcoe_wwn_from_mac(unsigned char mac[], unsigned int, unsigned int);
 int fcoe_libfc_config(struct fc_lport *, struct fcoe_ctlr *,
 		      const struct libfc_function_template *, int init_fcp);
 u32 fcoe_fc_crc(struct fc_frame *fp);
